### ISSUE TITLE
Request for detailed fine-tuning commands/parameters to reproduce reported results

### ISSUE THREAD

**Issue by ZhuYiyi-11** [ZiyangZhang0511/Echo-Vison-FM#3]
Thank you for sharing this great work and making the code publicly available!

I am trying to reproduce the fine-tuning results reported in your paper/README, specifically for the STF model on echonet_dynamic dataset.

Currently, I followed the command provided in the README:
`accelerate launch --config_file ./config_twogpus.yaml train_main.py [-option]`

However, my results are significantly lower than the reported ones. Could you kindly provide:

1. The complete command line with all arguments used for STF fine-tuning
2. Key hyperparameters (learning rate, epochs, batch size, etc.)
3. Any specific configurations needed for echonet_dynamic.

 This would greatly help the community to fairly compare and build upon your excellent work. 

Thank you very much for your time and assistance!

**Comment by Sean-C71** [comment]
Hi, I got the same question.
Have you successfully tuned the hyperparameters?

**Comment by ZhuYiyi-11** [comment]
> Hi, I got the same question. Have you successfully tuned the hyperparameters?

I’m afraid I haven’t solved it yet — still working on tuning the hyperparameters. T_T

**Comment by wengchunlei** [comment]
@Sean-C71 @ZhuYiyi-11  hello , Have you fixed the hyperparameters?

**Comment by wengchunlei** [comment]
I meet the same question,  need some help

**Comment by Sean-C71** [comment]
> [@Sean-C71](https://github.com/Sean-C71) [@ZhuYiyi-11](https://github.com/ZhuYiyi-11) hello , Have you fixed the hyperparameters?

Still haven't.

**Comment by fanjly** [comment]
请问echonet_dynamic数据集数据集在哪可以下载？

**Comment by ZhuYiyi-11** [comment]
> 请问echonet_dynamic数据集数据集在哪可以下载？

可以在数据集官网下载，网址如下：https://echonet.github.io/dynamic/index.html#dataset

**Comment by fanjly** [comment]
谢谢
